zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/UserTaskController.java

```diff
@@ -120,4 +130,34 @@ private static ProblemDetail mapRejectionToProblem(final BrokerRejection rejecti
         }
     };
   }
+
+  @PostMapping(
+      path = "/user-tasks/search",
+      produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_PROBLEM_JSON_VALUE},
+      consumes = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<UserTaskSearchQueryResponse> searchUserTasks(
+      @RequestBody(required = false) final UserTaskSearchQueryRequest query) {
+    return SearchQueryRequestMapper.toUserTaskQuery(query == null? new UserTaskSearchQueryRequest() : query)
```

Comment:
🔧 You could also allow to pass a `null` into the `toUserTaskQuery` method and handle it there gracefully. Just as an alternative.

Reply:
Thanks for the suggestion, I have changed this over the last commit.